### PR TITLE
Fixed: Return integer values when 'round to nearest integer' is selected in Group tool

### PR DESCRIPTION
**Pull request by natefoo**
From #425.

I wanted to get this fix in to 15.07 and since the freeze is today I went ahead and merged in @moskalenko's change locally and applied the correction from @nsoranzo.

Close #425 if merged.

### CODE REVIEW

**Comment by dannon** [comment]
Could probably drop the round() completely -- it shouldn't change anything here.  

For example, round() will drop 1.4 to 1.0 (still a float), then cast to int, 1.

```
In [1]: i = 1.4

In [2]: round(i)
Out[2]: 1.0

In [3]: int(i)
Out[3]: 1

In [4]: int(round(i))
Out[4]: 1
```

EDIT:  I was wrong -- obviously it rounds up, which integer casting does not.  Ignore me :)

**Comment by dannon** [comment]
+1

File: tools/stats/grouping.py [hunk 1]
```diff
@@ -148,10 +148,10 @@ def is_new_item(line):
                     sys.exit( 1 )
                 rval = getattr(numpy, op)( data )
                 if round_val[i] == 'yes':
-                    rval = round(rval)
+                    rval = int(round(rval))
                 else:
                     rval = '%g' % rval
-                        
+                rval = rval
```
Comment (nsoranzo) [inline tools/stats/grouping.py:154]: This line can be removed.

Comment (natefoo) [inline tools/stats/grouping.py:154]: Indeed... wow.

**Pull request merged**